### PR TITLE
avm2: Convert Font class to ActionScript

### PR DESCRIPTION
**Pull request by Aaron1011**

### CODE REVIEW

File: core/src/avm2/globals/flash/text/Font.as [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package flash.text {
+    public class Font {
+        public function Font() {}
```
Comment (Lord-McSweeney) [inline core/src/avm2/globals/flash/text/Font.as:3]: I don't think this constructor is necessary.

File: core/src/avm2/globals/Class.as
Comment (evilpie) [inline core/src/avm2/globals/Class.as:1]: This seems unrelated.

Comment (Aaron1011) [reply core/src/avm2/globals/Class.as:1]: This is needed by `Font.as`

**Review by Lord-McSweeney — APPROVED** [review]

**Review by Herschel — APPROVED** [review]
Thank you!
